operator/Makefile

```diff
@@ -176,8 +176,7 @@ build-installer: manifests generate kustomize ## Build consolidated installer YA
 ##@ Helm Chart - Deployment
 .PHONY: prepare-crd-chart
 prepare-crd-chart: manifests kustomize helmify ## Build CRDs using helmify.
-	$(KUSTOMIZE) build config/default | $(HELMIFY) ${CHART_PATH_CRD}
-	find ${CHART_PATH_CRD}/templates/ -type f ! \( -name '*-crd.yaml' -o -name '_helpers.tpl' \) -exec rm -f {} +
```

Comment:
This ensures we do not copy files that are not CRDs. Is there a reason to remove it?

Reply:
I'm using Kustomize itself to target the `config/crd` kustomization folder (not `config/default`) which therefore only output the CRDs, avoiding the need to delete extra YAML files.